MediaGalleryUi/etc/di.xml

```diff
@@ -26,6 +26,7 @@
                 <item name="theme" xsi:type="string">/pub\/media\/theme/</item>
                 <item name="theme_customization" xsi:type="string">/pub\/media\/theme_customization/</item>
                 <item name="tmp" xsi:type="string">/pub\/media\/tmp/</item>
+                <item name="directories-with-dots" xsi:type="string">/\./</item>
```

Comment:
@Nazar65 we need to exclude only directories that have the name starting with dot. Please take a look on my suggestion
```suggestion
                <item name="directories-with-dots" xsi:type="string">/^\./</item>
```